setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import find_packages, setup
+
+setup(
+    name="cytocherrypick",
+    version="0.1",
+    packages=find_packages(),
+    install_requires=[
```

Comment:
Consider updating this line to reference the `requirements.txt` file to source this information from a single place (where otherwise certain updates may be forgotten in one or the other location). See the following [example from Pycytominer](https://github.com/cytomining/pycytominer/blob/c4de0a9a4fecbf1ad11872bb14c18d24d1b1851e/setup.py#L28C22-L28C35).

Reply:
addressed